service/dap/handles.go

```diff
@@ -0,0 +1,33 @@
+package dap
+
+// Based on
+// https://github.com/microsoft/vscode-debugadapter-node/blob/master/adapter/src/handles.ts
+
+const startHandle = 1000
+
+type handles struct {
```

Comment:
No comments here?

"Based on ..." a TS file with no comments whatsoever :-) Better to add real comments to this type (+ some basic tests!), and not mention the adapter code

I'd also propose a more descriptive name like `uniqueHandleMap`.